apps/vr-tests-react-components/src/stories/Combobox.stories.tsx

```diff
@@ -88,13 +88,6 @@ storiesOf('Combobox Converged', module)
       <Option>text</Option>
     </Combobox>
   ))
-  .addStory('With multiselect value', () => (
-    <Combobox multiselect selectedOptions={['Green', 'Red']}>
-      <Option>Red</Option>
-      <Option>Green</Option>
-      <Option>Blue</Option>
-    </Combobox>
-  ))
```

Comment:
Removed this because Combobox doesn't render multiselect text into the value anyway. I added a multiselect screenshot when open instead.